[#25] move external url from `concourse-ci.org` to `pivotal.io`

YoussB opened the pull request:
All tasks from #25 have been completed.

- Changed from concourse-ci.org to pivotal.io
- Updated chart to use the latest helm repo
- removed requirements.lock file and added it to `.gitignore`
- Added web annotation to force recreate web pods on chart change

cc @taylorsilva

cirocosta commented on deployments/with-creds/Makefile:
```diff
@@ -30,6 +30,7 @@ diff-%: | ensure-creds-file-exists-% deps-%
 			--namespace=$* \
 			--detailed-exitcode \
 			--values=./.values.yaml \
+			--set=concourse.web.annotations.time=`date +%s` \
```
Hey,

Does this mean that even when there are no changes (both in terms of secrets and other configs) we end up updating the pods?

taylorsilva replied:
Yes we would, if you tried deploying with this change it would update all the pods. Why would one ever do a no-op change though?

I do think that this is a _clever_ workaround ([from here](https://github.com/helm/helm/issues/5218)) and makes it non-obvious what the behaviour is going to be. We actually couldn't find anywhere in the k8s docs (maybe we should've been looking at helm? unclear to me who's feature this is) that indicated that changing only annotations causes pods to be restarted.

We used this workaround because updating only the secrets resulted in just the secrets on the pods being remounted. Processes weren't being restarted, therefore changes weren't picked up :(

YoussB replied:
We removed this from the Makefile and added it to the `values.yaml` and added a note to the README about this annotation. We also renamed the annotation to `rollingUpdate`. See new commit for full details.

cirocosta commented on deployments/with-creds/hush-house/requirements.lock:
```diff
@@ -1,6 +0,0 @@
-dependencies:
```
Although this was indeed referencing an old version, I think it's still important to keep lock files as they list the digests and creation timestamps of our direct dependencies 🤔

YoussB replied:
We undid this change

cirocosta reviewed (changes requested):
Looks good except two points that might be worth discussing a bit 😁

taylorsilva committed:
[#25]

- Update Domain from concourse-ci.org to pivotal.io
- Update chart to use the latest helm repo
- Add web annotation to force recreate web pods on change

Signed-off-by: Taylor Silva <tsilva@pivotal.io>
Co-authored-by: Bishoy Youssef <byoussef@pivotal.io>

taylorsilva committed:
[#25]

- Add rollingUpdate annotation to worker and web deployments so we can
do rolling updates only when we want
- Remove annotation setting from the Makefile
- Add note about new annotation in README

Signed-off-by: Taylor Silva <tsilva@pivotal.io>
Co-authored-by: Bishoy Youssef <byoussef@pivotal.io>
Signed-off-by: Taylor Silva <tsilva@pivotal.io>

cirocosta reviewed (approved):
SGTM 😁  Thanks!
